src/main/java/net/honux/chess/GameMain.java

```diff
@@ -0,0 +1,29 @@
+package net.honux.chess;
+
+import java.nio.channels.ClosedByInterruptException;
+import java.util.Scanner;
+
+public class GameMain {
+    public static void main(String[] args) {
+        System.out.println("> 체스 게임을 시작하시려면 start, 끝내시려면 end 명령을 입력하세요.");
+        Board board = new Board();
+
+        String command;
+        try (Scanner sc = new Scanner(System.in);) {
+            while (true) {
+                command = sc.nextLine();
+                if (command.equals("end")) {
+                    System.out.println("Bye~");
+                    break;
+                }
+
+                if (command.equals("start")) {
+                    board.initialize();
+                    System.out.println(board.boardPrint());
+                }
+            }
+        } catch (Exception e) {
+            System.out.println(e);
+        }
```

Comment:
```suggestion
        try (Scanner sc = new Scanner(System.in)) {
            String command;
            while (true) {
                command = sc.nextLine();
                if (command.equals("end")) {
                    System.out.println("Bye~");
                    break;
                }

                if (command.equals("start")) {
                    board.initialize();
                    System.out.println(board.boardPrint());
                }
            }
        }
```

이렇게도 고칠 수 있을 것 같아보이는데 한 번 확인해주세요!